### ISSUE TITLE
Support Amazon S3 accesspoints

### ISSUE THREAD

**Issue by jakepage91** [tailwarden/komiser#579]
**Describe the solution you'd like**
Collect Amazon S3 accesspoints
**Additional context**
To get an idea of how to implement the collector, take the [S3 buckets](https://github.com/tailwarden/komiser/blob/master/providers/aws/s3/buckets.go) resource as inspiration.

**Comment by aimanafzal** [comment]
Hi @jakepage91 , Let me work on this.

**Comment by jakepage91** [comment]
For sure! Yours.

**Comment by aimanafzal** [comment]
> 

Thanks 👍

**Comment by ShubhamPalriwala** [comment]
Hey @aimanafzal, would you like to continue working on this now since the ECS Task Definition PR is almost ready to be merged 😉 ?

**Comment by aimanafzal** [comment]
Yes, please open this issue too :)

Both could be done at almost the same time.

On 07-Jul-2023, at 10:26 PM, Shubham Palriwala ***@***.***> wrote:



Hey @aimanafzal<https://github.com/aimanafzal>, would you like to continue working on this now since the ECS Task Definition PR is almost ready to be merged 😉 ?

**Comment by ShubhamPalriwala** [comment]
Hey, the issue is open and is already assigned to you! Please open a PR whenever you are ready with it, looking forward to it 😄

**Comment by aimanafzal** [comment]
> Hey, the issue is open and is already assigned to you! Please open a PR whenever you are ready with it, looking forward to it 😄

Starting to work on it

**Comment by aimanafzal** [comment]
> > Hey, the issue is open and is already assigned to you! Please open a PR whenever you are ready with it, looking forward to it 😄
> 
> Starting to work on it

@ShubhamPalriwala I don't see the service in the list. Even though I have got the policy updated in my AWS.
Is there anything that I am doing wrong?

**Comment by aimanafzal** [comment]
> > > Hey, the issue is open and is already assigned to you! Please open a PR whenever you are ready with it, looking forward to it 😄
> > 
> > 
> > Starting to work on it
> 
> @ShubhamPalriwala I don't see the service in the list. Even though I have got the policy updated in my AWS. Is there anything that I am doing wrong?

@jakepage91 @ShubhamPalriwala

**Comment by ShubhamPalriwala** [comment]
Hey @aimanafzal, I couldn't understand you! are you asking about where to find the Access Points? Does this help then:
 https://aws.amazon.com/s3/features/access-points/?

**Comment by aimanafzal** [comment]
> Hey @aimanafzal, I couldn't understand you! are you asking about where to find the Access Points? Does this help then: https://aws.amazon.com/s3/features/access-points/?

No, it is not that. I mean, on the Komiser screen. It is not showing up. 
I have checked the policy and it should be fine.

**Comment by aimanafzal** [comment]
Yes, there is.

On 14-Aug-2023, at 5:58 PM, LABOUARDY Mohamed ***@***.***> wrote:



Hey @aimanafzal<https://github.com/aimanafzal>, I couldn't understand you! are you asking about where to find the Access Points? Does this help then: https://aws.amazon.com/s3/features/access-points/?

No, it is not that. I mean, on the Komiser screen. It is not showing up. I have checked the policy and it should be fine.

do you've any S3 access points created in your AWS account? what's the output of this command<https://docs.aws.amazon.com/cli/latest/reference/s3control/list-access-points.html>.

**Comment by jakepage91** [comment]
Hey there @aimanafzal I really apologize! We seem to have lost the thread here on this issue in the last few weeks! I wanted to revisit this to see exactly where you got stuck. Could give me a refresher? So I can best help you out.

**Comment by aimanafzal** [comment]
Hey @jakepage91, Thanks for reaching out. 

The issue is, that I don't see the service in the list on my screen. Even though I have got the policy updated in my AWS.

**Comment by jakepage91** [comment]
Hey there @aimanafzal do you see any fetched S3 accesspoints in the CLI which then don't appear in the cloud inventory or you don't even see them being fetched through the CLI?

**Comment by aimanafzal** [comment]
Only S3 as a service is visible.

On 26-Oct-2023, at 3:08 PM, Jake Page ***@***.***> wrote:



Hey there @aimanafzal<https://github.com/aimanafzal> do you see any fetched S3 accesspoints in the CLI which then don't appear in the cloud inventory or you don't even see them being fetched through the CLI?

**Comment by aimanafzal** [comment]
@jakepage91 Did you checked?
